podcasts/EpisodeBookmarksTabsView.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+/// Displays a fake set of tabs that allows the user to open the bookmarks view from the podcast list
+struct EpisodeBookmarksTabsView: View {
```

Comment:
I didn't make this generic because of the way this "tab view" works where the episode "button" isn't real and tapping the bookmarks button doesn't change state or anything and just triggers a single action. 

For the episode detail I'll make something better.

Reply:
That's also why I am using the `PodcastActionsDelegate` directly.